src/picongpu/CMakeLists.txt

```diff
@@ -258,6 +266,7 @@ find_package(Boost 1.56.0 REQUIRED COMPONENTS program_options regex filesystem
                                               system thread math_tr1)
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 set(LIBS ${LIBS} ${Boost_LIBRARIES})
+message(STATUS "Boost libs = ${Boost_LIBRARIES}")
```

Comment:
not sure if we want this because usually CMake outputs this on its own. but if we want  to add it, use the commonly used cmake output syntax
```cmake
message(STATUS "Found Boost: ${Boost_LIBRARIES}")
```